platform/ios/src/MGLMapViewDelegate.h

```diff
@@ -247,7 +247,7 @@ NS_ASSUME_NONNULL_BEGIN
  @param annotation The annotation being rendered.
  @return A line width for the polyline, measured in points.
  */
-- (CGFloat)mapView:(MGLMapView *)mapView lineWidthForPolylineAnnotation:(MGLPolyline *)annotation;
+- (float)mapView:(MGLMapView *)mapView lineWidthForPolylineAnnotation:(MGLPolyline *)annotation;
```

Comment:
Thanks for the explanation @1ec5 - `PropertyValue<float>` doesn't like to convert from double, so I guess I'll need to add a static_cast in that case.